src/stylesheets/elements/_media.scss

```diff
@@ -7,19 +7,27 @@ video {
   width: 100%;
 }
 
+figure {
+  display: inline-block;
+  vertical-align: top;
+  width: 100%;
+}
+
 picture {
+  background-color: $grey-4;
   display: block;
+  min-height: 30px;
+  width: 100%;
 }
 
 img {
-  z-index: 1;
-  width: 100%;
+  color: $grey-4;
   display: block;
-  position: relative;
+  font-size: 0;
```

Comment:
What's with the font-size being set to 0?

Reply:
In some instances, images are within links and has the link underline decoration applied, this causes our fallback to be visually incorrect. Applying no text decoration to images does not work, therefore font size was set to 0 to remove the underline effect.